taxcalc/functions.py

```diff
@@ -824,7 +824,7 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, c24517,
         c62100 += _cmbtp
     if puf and _standard > 0:
         if f6251 == 1:
-            _cmbtp = _cmbtp_standard
+            _cmbtp = e62100 - e00100 + e00700
```

Comment:
@martinholmer, I believe AMT preferences (`_cmbtp`) ought to be imputed with the base year's data and then blown up separately. Therefore, moving the imputation for standard deduction filers functions.py here is not correct. I think we ought to treat _cmbtp like all of our other imputations: do the imputation in TaxData and then include the starting year values in puf.csv.